spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/SparkSQLProperties.java

```diff
@@ -47,4 +47,8 @@ private SparkSQLProperties() {}
   public static final String PRESERVE_DATA_GROUPING =
       "spark.sql.iceberg.planning.preserve-data-grouping";
   public static final boolean PRESERVE_DATA_GROUPING_DEFAULT = false;
+
+  // Controls write distribution mode
+  public static final String SESSION_WRITE_DISTRIBUTION_MODE =
```

Comment:
I don't think we need the `SESSION_` prefix in the name. All SQL properties are session properties.

Reply:
I see our SQL configs rarely include read/write prefixes. What about this then?

```
public static final String DISTRIBUTION_MODE = "spark.sql.iceberg.distribution-mode"
```